discussment-learning/src/main/java/org/danekja/discussment/learning/service/imp/DefaultArticleService.java

```diff
@@ -0,0 +1,42 @@
+package org.danekja.discussment.learning.service.imp;
+
+import org.danekja.discussment.core.domain.Discussion;
+import org.danekja.discussment.learning.dao.ArticleDao;
+import org.danekja.discussment.learning.domain.Article;
+import org.danekja.discussment.learning.service.ArticleService;
+import org.danekja.discussment.core.service.DiscussionService;
+
+import java.util.List;
+
+/**
+ * Implementation of the ArticleSevice interface.
+ *
+ * Date: 30.1.18
+ *
+ * @author Jiri Kryda
+ */
+public class DefaultArticleService implements ArticleService {
+    private ArticleDao articleDao;
+
+    public DefaultArticleService(ArticleDao articleDao){ this.articleDao = articleDao; }
```

Comment:
Make it take DiscussionService as constructor parameter and set an attribute of type DiscussionService.